mesonbuild/dependencies/dub.py

```diff
@@ -128,9 +128,9 @@ def __init__(self, name: str, environment: 'Environment', kwargs: T.Dict[str, T.
         # A command that might be useful in case of missing DUB package
         def dub_build_deep_command() -> str:
             if self._dub_has_build_deep:
-                cmd = [ 'dub', 'build', '--deep' ]
+                cmd = ['dub', 'build', '--deep']
```

Comment:
Changes like this, that fix a previous commit, should be created with `git commit --fixup=old-commit` and squashed during rebasing.